jest.config.js

```diff
@@ -12,6 +12,8 @@ module.exports = {
     '@openmrs/esm-framework': '@openmrs/esm-framework/mock',
     '^dexie$': require.resolve('dexie'),
     '^lodash-es/(.*)$': 'lodash/$1',
+    '^@tools/(.*)$': path.resolve(__dirname, 'tools', '$1'),
+    '^__mocks__/(.*)$': path.resolve(__dirname, '__mocks__', '$1'),
```

Comment:
Minor, but maybe we could use `@mocks` as the path alias instead of `__mocks__` for consistency?

Reply:
Done!